lusidtools/cocoon/utilities.py

```diff
@@ -1380,7 +1382,7 @@ def create_scope_id(time_generator=None):
         Scope identifier
     """
 
-    return generate_time_based_unique_id(time_generator)
+    return generate_time_based_unique_id(time_generator) + str(uuid.uuid4())
```

Comment:
Can this be used as-is in most operations? Or does it always need to be truncated to be used as an id in LUSID because it is too long? I believe the limit is 64 characters for scopes and codes.